tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.txt

```diff
@@ -0,0 +1,38 @@
+<Configuration> 
+ <TestRoot>..\..</TestRoot> 
+ <TargetArchitecture>x64</TargetArchitecture> 
+ <ScriptRootDir>$(TestRoot)\Scripts</ScriptRootDir> 
+ <DebuggeeRootDir>$(TestRoot)\Debuggees</DebuggeeRootDir> 
+ <DebuggeeSourceRoot>$(DebuggeeRootDir)\Source</DebuggeeSourceRoot>
+ <RuntimeRoot>##Insert_Runtime_Root##</RuntimeRoot>
+ <NugetCacheRoot>##Insert_Nuget_Cache_Root##</NugetCacheRoot>
+ <Options> 
+   <Option> 
+     <TestProduct>ProjectK</TestProduct> 
+     <HostExe>$(RuntimeRoot)\corerun.exe</HostExe> 
+     <DbgShim>$(RuntimeRoot)\dbgshim.dll</DbgShim> 
+     <RuntimeSymbolsPath>$(RuntimeRoot)</RuntimeSymbolsPath> 
+     <DebuggeeBuildProcess>cli</DebuggeeBuildProcess> 
+     <CliVersion>1.0.0-preview1-002702</CliVersion> 
+     <DebuggeeNativeLibRoot>$(DebuggeeRootDir)\native</DebuggeeNativeLibRoot> 
+     <DebuggeeBuildRoot>$(DebuggeeRootDir)\dotnet.$(CliVersion)</DebuggeeBuildRoot> 
+     <ProjectJsonNetStandardLibraryVersion>1.0.0-rc2-23811</ProjectJsonNetStandardLibraryVersion> 
+     <ProjectJsonSystemThreadingThreadVersion>4.0.0-rc2-24027</ProjectJsonSystemThreadingThreadVersion> 
+     <ProjectJsonMicrosoftCSharpVersion>4.0.1-rc2-23811</ProjectJsonMicrosoftCSharpVersion> 
+     <ProjectJsonRuntime>win7-x64</ProjectJsonRuntime> 
+     <CliPath>https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/$(CliVersion)/dotnet-dev-win-x64.$(CliVersion).zip</CliPath> 
```

Comment:
Do you want to download a new CLI from the internet? I think coreclr repo already downloaded one as part of its own setup.